pkg/cmd/server/start/start_master.go

```diff
@@ -395,18 +396,8 @@ func (m *Master) Start() error {
 		if err := startAPI(openshiftConfig, kubeMasterConfig); err != nil {
 			return err
 		}
-		if m.controllers {
-			// run controllers asynchronously (not required to be "ready")
-			go func() {
-				if err := startControllers(openshiftConfig, kubeMasterConfig); err != nil {
-					glog.Fatal(err)
-				}
-			}()
-		}
-		return nil
-	}
 
-	if m.controllers {
+	case m.controllers:
 		glog.Infof("Starting controllers on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
```

Comment:
The controllers print their own messages - this is the summarization message about what mode are you in.